cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -383,25 +393,29 @@ def test_no_power_controls_on_archived_vm(testing_vm, archived_vm, soft_assert):
     soft_assert(not view.toolbar.power.is_displayed, "Power displayed in archived VM's details!")
 
 
+@pytest.mark.rhv3
 @pytest.mark.uncollectif(lambda provider: provider.one_of(SCVMMProvider) and
                          BZ(1520489, forced_streams=['5.9']).blocks, 'BZ 1520489')
 def test_archived_vm_status(testing_vm, archived_vm):
     vm_state = testing_vm.find_quadicon(from_any_provider=True).data['state']
     assert ('currentstate-archived' in vm_state)
 
 
+@pytest.mark.rhv3
 def test_orphaned_vm_status(testing_vm, orphaned_vm):
     vm_state = testing_vm.find_quadicon(from_any_provider=True).data['state']
     assert ('currentstate-orphaned' in vm_state)
 
 
+@pytest.mark.rhv3
 @pytest.mark.uncollectif(lambda provider: provider.one_of(RHEVMProvider))
 def test_power_options_from_on(provider, soft_assert, testing_vm, verify_vm_running):
     testing_vm.wait_for_vm_state_change(
         desired_state=testing_vm.STATE_ON, timeout=720, from_details=True)
     check_power_options(provider, soft_assert, testing_vm, testing_vm.STATE_ON)
 
 
+@pytest.mark.rhv3
```

Comment:
There is maybe an arguemnt for this and test_power_options_from_on to move to higher tier, since this could seriously inhibit the use of CFME with RHV. But I don't say it's wrong, just food for thought

Reply:
What about this, @istein1?

Reply:
* test_power_options_from_off is marked with T3. We have test_vm_power_control.py::TestControlOnQuadicons::test_power_on marked as T1, and covers  the power on - the only valid operation if VM is down.

* test_power_options_from_on (power off, shutdown, restart, suspend)- This test is currently skipped for RHV. 
I need to open a ticket on it. I think it is because we are missing the "reset" option for RHV (not supported).
power off & suspend UI tests, are marked with T1, however I do not see the shutdown & restart options mentioned in any power tests.
I shall mark this test as T1, and further investigation on the missing options will continue in such ticket.